test/integration_tests/abao/load_fixture.py

```diff
@@ -0,0 +1,444 @@
+    # depends on 'create session 1 test-analysis (file upload)'
+    r = session.post('/sessions/' + test_session['_id'] + '/analyses/' + test_session_analysis['_id'] + '/notes', json={
+        'text': 'test note'
+    })
+    assert r.ok
+
+    # add a note to test-project-1 test-analysis-1
+    # depends on 'create project 1 test-analysis (file upload)'
+    r = session.post('/projects/' + test_project['_id'] + '/analyses/' + test_project_analysis_upload['_id'] + '/notes', json={
+        'text': 'test note'
+    })
+    assert r.ok
+
+    # create project
+    r = session.post('/projects', json={
+        'group': 'test-group',
+        'label': 'Project with template',
+        'public': False
+    })
+    assert r.ok
+    st_project = r.json()
+
+    # create compliant session
+    # depends on 'create project'
+    r = session.post('/sessions', json={
+        'subject': { 'code': 'ex8945' },
+        'label': 'Compliant Session',
+        'project': st_project['_id'],
+        'public': False
+    })
+    assert r.ok
+    st_compliant_session = r.json()
+
+    # create non-compliant session
+    # depends on 'create project'
+    r = session.post('/sessions', json={
+        'subject': { 'code': 'ex9849' },
+        'label': 'Non-compliant Session',
+        'project': st_project['_id'],
+        'public': False
+    })
+    assert r.ok
+    st_noncompliant_session = r.json()
+
+    # create acquisition-1 for compliant session
+    # depends on 'create compliant session'
+    r = session.post('/acquisitions', json={
+        'label': 'c-acquisition-1-t1',
+        'session': st_compliant_session['_id'],
+        'public': False
+    })
+    assert r.ok
+    st_cs_acquisition_1 = r.json()
+
+    # create acquisition-2 for compliant session
+    # depends on 'create compliant session'
+    r = session.post('/acquisitions', json={
+        'label': 'c-acquisition-2-t1',
+        'session': st_compliant_session['_id'],
+        'public': False
+    })
+    assert r.ok
+    st_cs_acquisition_2 = r.json()
+
+    # create acquisition-1 for noncompliant session
+    # depends on 'create non-compliant session'
+    r = session.post('/acquisitions', json={
+        'label': 'nc-acquisition-1-t1',
+        'session': st_noncompliant_session['_id'],
+        'public': False
+    })
+    assert r.ok
+    st_ncs_acquisition_1 = r.json()
+
+    # add project template
+    r = session.post('/projects/' + st_project['_id'] + '/template', json={
+        'session': { 'subject': { 'code' : '^ex' } },
+        'acquisitions': [{
+            'label': 't1',
+            'minimum': 2
+        }]
+    })
+    assert r.ok
+    assert r.json()['modified'] == 1
+
+    # test session 1 is marked as compilant
+    # depends on 'create compliant session'
+    r = session.get('/sessions/' + st_compliant_session['_id'])
+    assert r.ok
+    assert r.json()['project_has_template'] == True
+    assert r.json()['satisfies_template'] == True
```

Comment:
The coverage was already "duplicated" in python tests, so these below should definitely go away as they aren't needed. Thanks!